letsencrypt/Dockerfile

```diff
@@ -1,15 +1,33 @@
 ARG BUILD_FROM
 FROM $BUILD_FROM
 
-# Setup base
+# setup base
 ARG CERTBOT_VERSION
-RUN apk add --no-cache \
-        openssl libffi musl libstdc++ \
+
+RUN apk add --no-cache --update \
+        openssl libffi musl libstdc++ wget tar python python-dev py-pip gcc \
```

Comment:
`libstdc++ wget tar python python-dev py-pip gcc` is part of our base image and can be removed